src/version.rs

```diff
@@ -1,32 +0,0 @@
-//! RPM versions
-
-use std::fmt::{self, Display};
-
-/// Package versions
-// TODO: `Ord` (i.e. dependency sets)
```

Comment:
Likewise the version string isn't sufficient to do "package version" comparisons, you need the epoch and release.

Reply:
Sounds like it'd be good to add an `Evr` type then, which could make that possible, although I'd suggest exploring that in a separate PR.

Reply:
There's one in librpm that is used as an argument for a bunch of important functions (including the RPM version comparisons) https://github.com/rpm-software-management/rpm/blob/4afe2d14d33db82ccb41c0a8d5eb1a4db90762fc/rpmio/rpmver.c#L9-L14